resources/content/collections/brands.yaml

```diff
@@ -1,8 +1,14 @@
-title: 'Brand content'
+title: Brands
```

Comment:
We called it "Brand content" just like the categories and products as there is also a Raleway resource. Otherwise we've 2x brands.